src/server/middlewares/logger.middleware.js

```diff
@@ -0,0 +1,14 @@
+import logsService from '../services/logs.service';
+
+const loggerMiddleware = (req, res, next) => {
+  const route = req.url;
+  const time = Date.now();
+  const { method } = req;
+  const body = method === 'GET' || method === 'DELETE' || route === '/api/login' || route === '/api/register' ? '' : JSON.stringify(req.body);
```

Comment:
It would look better if we create a separate constant with a list of routes that should be ignored by logger and then check if a route is in this list.